### DISCUSSION TITLE
Triggering external webhook when rowy table updates?

### DISCUSSION THREAD

**Discussion by Settld-Admin** [AntlerVC/firetable#766] [Support / Q&A] [answered]
Is there a (nocode/lowcode) method to have a table in rowy trigger an external webhook (e.g one created in make.com) each time the table is updated?

**Comment by harinij** [comment, marked as answer]
@Settld-Admin We do not have a live example for this but you can do this using the generic task ["extension"](https://docs.rowy.io/category/extensions) using which you can any external API anytime a row in a table is added/updated/deleted. If you are looking for getting data into rowy using an external webhook, you can do so using the webhook feature of Rowy, we have a few examples [here](https://docs.rowy.io/category/webhooks).
